### DISCUSSION TITLE
Can't add linked fields in admin view

### DISCUSSION THREAD

**Discussion by hdwverden** [joomengine/Joomla-Component-Builder#858] [Extention Issues] [answered]
I just inherited a project built in Component Builder with 197 linked fields in admin view (I know this is bad practice). When trying to add even one more linked field instead of successfully saving it just loads the Component Builder Dashboard.

I already tried building a new Admin View from scratch, however in the new one I can’t add more than 128 fields. Is there any way around that limit?

It is a contact manager and there are lots of committees and qualifications each contact can have so every qualification etc has to be its own radio button.

Thanks in advance.

**Comment by SimonvanDoorne** [comment]
Is it possible for you to change some of the fields to subforms? You'd have to change the DB layout a bit but that would at least help with the field limit. Otherwise maybe the field definition JCB uses for the linked fields themself has a max limit worth a shot looking if that can be changed.

**Comment by Llewellynvdm** [comment, marked as answer]
Okay believe it or not, the limitation is not in JCB but a PHP limitation, so to be able to add more fields you need to [increase the following values](https://stackoverflow.com/a/9399708/1429677) in you PHP.ini file.
```ini
max_input_time = 1300
max_input_vars = 2000
```
Just in the local development... since when you add that many fields to a view, JCB has to post all that values, and that is where it breaks when it hits up against these limits.

**Reply by hdwverden** [reply]
Thanks, that did the trick.
